about.html

```diff
@@ -81,57 +87,63 @@ <h2>South Korea</h2>
             
-            <div class="card text-center p-2 hero-image text-light mb-2 tw-400">
-                <div class="card-body">
-                  <h3 class="card-title">2019</h3>
-                  <h6 class="card-subtitle mb-2">Winner: Japan</h6>
-                  <p class="card-text">Location: Brazil</p>
+            <div class="presenters">
+                <div class="card text-center hero-image text-light mb-2 tw-400">
+                    <div class="card-body hero-div">
+                      <h3 class="card-title mt-2">2019</h3>
+                      <h6 class="card-subtitle mb-2">Winner: Japan</h6>
+                      <p class="card-text mb-2">Location: Brazil</p>
+                    </div>
                 </div>
-            </div>
-
-            <div class="card text-center p-2 hero-image text-light mb-2 tw-400">
-                <div class="card-body">
-                  <h3 class="card-title">2018</h3>
-                  <h6 class="card-subtitle mb-2">Winner: Japan</h6>
-                  <p class="card-text">Location: Brazil</p>
+    
+                <div class="card text-center hero-image text-light mb-2 tw-400">
+                    <div class="card-body hero-div">
+                      <h3 class="card-title mt-2">2019</h3>
+                      <h6 class="card-subtitle mb-2">Winner: Japan</h6>
+                      <p class="card-text mb-2">Location: Brazil</p>
+                    </div>
                 </div>
             </div>
+            
 
         </div>
 
         <!--div 4 dark-->
         <div class="bg-dark text-light text-center">
             <br>
             <h2>Partner</h2>
-            <hr class="my-4 text-primary bg-primary border-5">
+            <hr>
             <div class="d-flex flex-wrap justify-content-center">
                 <h2 class="m-2">mozilla</h2>
                 <h2 class="m-2">Microsoft</h2>
                 <h2 class="m-2">Google</h2>
                 <h2 class="m-2">Alex Dev</h2>
                 <h2 class="m-2">Udemy</h2>
             </div>
+            <br>
         </div>
 
         <br>
         <!--footer light-->
         <footer class="bg-light text-dark">
-            <div class="d-flex flex-row p-1">
-                <div class="me-2">
+            <div class="footer-info">
+                <div class="me-2 footer-logo">
                     <img class="me-1 logo" src="./assets/Logo.svg" alt="">
                 </div>
                 <div>
-                    <p><small>2021 Alex Dev Mexico.</small></p>
-                    <p><small>All Rights Reserved</small></p>
+                    <p><small>2021 Alex Dev Mexico. All Rights Reserved</small></p>
+                    <p class="mobile-hidden"><small>Alex Dev company, is a non-profit company, for any purpose related to this, please contact wikitelmex@gmail.com</small></p>
                 </div>
             </div>
+            br
         </footer>
```

Comment:
- [ ] Please, consider make this for the footer: 🦊 
![image](https://user-images.githubusercontent.com/20563252/132961497-beeba74f-1cc9-43c6-8324-acd2a47457a8.png)